core/src/main/java/org/apache/spark/shuffle/api/ShuffleExecutorComponents.java

```diff
@@ -39,17 +39,15 @@
   /**
    * Called once per map task to create a writer that will be responsible for persisting all the
    * partitioned bytes written by that map task.
-   *  @param shuffleId Unique identifier for the shuffle the map task is a part of
-   * @param mapId Within the shuffle, the identifier of the map task
-   * @param mapTaskAttemptId Identifier of the task attempt. Multiple attempts of the same map task
- *                         with the same (shuffleId, mapId) pair can be distinguished by the
- *                         different values of mapTaskAttemptId.
+   * @param shuffleId Unique identifier for the shuffle the map task is a part of
+   * @param mapId Identifier of the task attempt. Multiple attempts of the same map task with the
```

Comment:
let's rephrase it. How about `An id of the map task which is unique within this Spark application.`

Reply:
Sure, done in 4bd9e00.